build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ -f pvs-*.tar.gz ]; then 
+	rm -f pvs-*.tar.gz
+fi
+
+tmp_ver=($(cat ./pvs/default/app.conf | grep version | tr "=" " "))
```

Comment:
Is pvs the correct config path and filename here?

Reply:
Yes, as we have to leave the app folder name as pvs so that it updates the old app anyone is using.